### PR TITLE
Tappable editable row

### PR DESCRIPTION
**Pull request by Eddie-42**
As suggested in the user study, where almost everyone expected the respective category titles to be tappable. I'm not quite sure yet if really the whole row should be tappable or if we should keep the icon buttons and just make the text clickable as well (like for the username). Feedback appreciated!
@Showcas This could have a slight impact on your tests.

### CODE REVIEW

**Comment by fynsta** [comment]
Another thing where I don't really agree with user studies, but implementation looks fine 👍

File: lib/account/pages/profile_page.dart [hunk 1]
```diff
@@ -330,6 +309,12 @@ class _ProfilePageState extends State<ProfilePage> {
     );
   }
 
+  Future<void> _pushAndLoadProfile(Widget page) async {
```
Comment (fynsta) [inline lib/account/pages/profile_page.dart:312]: the name is confusing - lt sounds like it is loading the profile immediately. Maybe "_pushEditPage", the exact implementation does not need to be exposed.

**Comment by fynsta** [comment]
Another thing where I don't really agree with user studies, but implementation looks fine 👍

**Comment by istzen** [comment]
i think if we make the whole row tappable then the text should also be tappable, how do you guys see this?

**Comment by Eddie-42** [comment]
> i think if we make the whole row tappable then the text should also be tappable, how do you guys see this?

I don't really know what you mean, you can tap the text and go to the page right?

**Comment by istzen** [comment]
> > i think if we make the whole row tappable then the text should also be tappable, how do you guys see this?
> 
> I don't really know what you mean, you can tap the text and go to the page right?

yes but if i tap for example the features i select nothing happens, that's what i meant but not sure if thats what we want

**Comment by fynsta** [comment]
I don't think that's expected by the user, is it?

**Comment by istzen** [comment]
i expected that that's why i mentioned it xD

**Comment by Eddie-42** [comment]
Oh I think there should be space in our app for the user to comfortably rest their hand without anything happening :D Meaning I think not vitually the whole page should be tappable. Did anyone see anyone in the user studies tap on the actual values? Because I and everyone I tested with only tried to tap the title

**Comment by Showcas** [comment]
I think it looks good, I would for safe usage only leave it to the text and the button. If they tap by accident on free space then they are probably confused when they get to the edit page every time

**Comment by Showcas** [comment]
Should I do the tests then in a seperate PR? or do you want to do it in yours since the profile tests are already merged

**Comment by fynsta** [comment]
Look at the diff, the tests are already changed

File: lib/account/pages/profile_page.dart [hunk 2]
```diff
@@ -119,18 +119,14 @@ class _ProfilePageState extends State<ProfilePage> {
     final Widget description = _profile!.description?.isNotEmpty ?? false
         ? Text(
             _profile!.description!,
-            style: Theme.of(context).textTheme.bodyLarge,
+            style: Theme.of(context).textTheme.bodyLarge!.copyWith(fontWeight: FontWeight.normal),
```
[the remark this answers is not in the document]
Comment (fynsta) [reply lib/account/pages/profile_page.dart:122]: How would that look?

[the remark this answers is not in the document]
Comment (Eddie-42) [reply lib/account/pages/profile_page.dart:122]: image

[the remark this answers is not in the document]
Comment (fynsta) [reply lib/account/pages/profile_page.dart:122]: I agree that that's better

**Review by Showcas — APPROVED** [review]

**Review by istzen — APPROVED** [review]

**Review by fynsta — APPROVED** [review]

**Pull request merged**